pkgs/top-level/python-packages.nix

```diff
@@ -4364,7 +4364,9 @@ in {
 
   nbconflux = callPackage ../development/python-modules/nbconflux { };
 
-  nbconvert = callPackage ../development/python-modules/nbconvert { };
+  nbconvert = callPackage ../development/python-modules/nbconvert {
+    nbclient = self.nbclient.override { doCheck = false; };
```

Comment:
This is not correct, because it can result in the same package ending up in a closure twice. Note the original change was incorrect as well; tests will need to be disabled entirely for a package. An exception could be when it is a build/test time dependency only.

Reply:
Note in practice it should cause no harm; its the exact same package after all.

Reply:
It was introduced in https://github.com/NixOS/nixpkgs/commit/113083c7ebb7bb1921ea0195f9e0661e657fda7f by you and jonringer. I am not sure how to fix this properly. If you can give me some hint I could try to fix this.

Reply:
Tests need to be disabled for nbclient, always. cc @jonringer

Reply:
I'm okay with that

Reply:
was trying to do the pytest / hypothesis DAG, however, they just use each other for test dependencies, so they don't leak. Yea, I should just have always disabled tests.